### PR TITLE
feat: add dependabot

### PR DESCRIPTION
**Pull request by DevHaoZi**
Close goravel/goravel#140

### CODE REVIEW

**Comment by hwbrzzl** [comment]
Hi @DevHaoZi The IoC feature has been merged to master, so would we like to deploy this PR to master or v1.11.x? If we want to use this PR fast, we can create a PR based on v1.11.x branch(now is master).

**Comment by DevHaoZi** [comment]
> The IoC feature has been merged to master, so would we like to deploy this PR to master or v1.11.x? If we want to use this PR fast, we can create a PR based on v1.11.x branch(now is master).

I think we'd better merge this into 1.12.x because this might introduce some "breaking changes".

File: event/application.go [hunk 1]
```diff
@@ -25,7 +25,7 @@ func (app *Application) GetEvents() map[event.Event][]event.Listener {
 }
 
 func (app *Application) Job(event event.Event, args []event.Arg) event.Task {
-	listeners, _ := app.events[event]
+	listeners := app.events[event]
 
 	return NewTask(app.queue, args, event, listeners)
 }
```
Comment (hwbrzzl) [inline event/application.go:31]: `listeners := app.events[event]` will be panic, we can change this like below:
```
func (app *Application) Job(e event.Event, args []event.Arg) event.Task {
	listeners, ok := app.events[e]
	if !ok {
		listeners = make([]event.Listener, 0)
	}

	return NewTask(app.queue, args, e, listeners)
}
```

**Review by hwbrzzl — APPROVED** [review]
LGTM

**Pull request merged**